Apps/Sandcastle/gallery/Particle System.html

```diff
@@ -398,6 +400,11 @@
 });
 
 var options = [{
+    text : 'Sphere',
+    onselect : function() {
+        particleSystem.emitter = new Cesium.SphereEmitter(2.5);
+    }
+}, {
     text : 'Circle',
     onselect : function() {
         particleSystem.emitter = new Cesium.CircleEmitter(0.5);
```

Comment:
Do any of these other emitters need tweaks to their size to look good with the changes>